lib/tasks/licenses.rake

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+namespace :licenses do
+
+  report_path = Rails.root.join('doc/licenses/licenses.xml')
+
+  desc 'Generates a report with the dependencies and their licenses'
+  task :report do
+    $stdout.puts 'Generating report...'
+    exec license_finder_report
+  end
+
+  desc 'Check license compliance of dependencies'
+  task :compliance do
+    $stdout.puts 'Checking license compliance...'
+    unless system('license_finder action_items')
+      $stderr.puts '*** License compliance test failed  ***' # rubocop:disable Style/StderrPuts
+      exit 1
+    end
+
+    $stdout.puts 'Checking license report is up to date...'
```

Comment:
>  I see that rubocop recommends to use warn "message" instead of $stderr.puts. So maybe use that instead?
😆 
https://github.com/3scale/porta/pull/3093#discussion_r1008057770

Reply:
I didn't know it will complain :man_facepalming:

Reply:
But don't we want to present the string 'Checking license report is up to date...' to the user? otherwise the task would have no standard output. I agree not everything in `stderr` must be an error, but in this case, if I run the task and redirect the `ERR` output to a file or `/dev/null`, I won't see anything.

Reply:
I agree with both but since these messages are meant for us developer and they really don't give much value... I'd leave it in `stderr` if not remove them altogether.

Reply:
@jlledom , stderr is still presented to user. The point is that in shell, the data that one would pipe into simething else goes to stdout. And any other information messages should go to stderr. I don't know of a formal standard but this is what I have observed. So stderr is still presented to user.